filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -937,8 +937,36 @@ void OpenGLDriver::framebufferTexture(backend::TargetBufferInfo const& binfo,
         attachmentTypeNotSupportedByMSRTT = true;
     }
 
-    if (rt->gl.samples <= 1 ||
-        (rt->gl.samples > 1 && t->samples > 1 && gl.features.multisample_texture)) {
+    // The following 4 techniques are different mechanisms to provide storage to the FBO.
+    // We'll use the first technique that is sufficient.
+    // nonMSAA:         MSAA was not requested, or we're rendering to an MS texture which
```

Comment:
maybe instead of "nonMSAA" it should be more like "noAutomaticMsaaResolve"? i.e. We won't do an automatic MSAA resolve (because either we don't have MSAA at all, or the attachment itself is MS and the resolve will have to be custom). Either way, we're not handling it. 

*However* because MS textures are not supported in ES3, we just do nothing. Meaning we effectively disable MSAA. This is *wrong* if we wanted to support ES3.1 MS textures.